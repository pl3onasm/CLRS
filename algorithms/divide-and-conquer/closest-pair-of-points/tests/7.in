1000
(197.756295,749.778914),(22.139362,708.572808),(888.044838,387.728970),(273.623293,947.650821),(203.459450,174.591682),(952.016456,974.502832),(655.366153,954.037252),(465.097354,241.907476),(337.920611,648.909451),(801.462248,735.647417),(173.522630,885.052034),(394.422300,161.167927),(227.831259,154.783833),(493.626680,465.725022),(733.236282,242.190976),(34.327358,930.992577),(991.969890,56.466721),(639.565385,880.014728),(444.195691,913.188678),(827.665549,647.655141),(87.780359,779.682005),(622.157973,743.146513),(733.719256,87.255326),(985.053989,71.639867),(736.164777,786.516237),(807.287284,909.687408),(671.568270,201.709584),(70.855334,899.399529),(356.493418,564.482015),(365.124550,89.729700),(806.672991,399.451909),(20.722277,798.642881),(455.918629,660.287662),(678.657608,900.114320),(573.476340,506.323156),(547.769461,661.256699),(286.005160,169.927433),(404.403211,19.724416),(257.182759,389.457200),(91.364283,993.347537),(175.973436,898.651567),(903.034945,847.541707),(100.361151,973.890279),(746.941235,456.854569),(538.372294,112.065784),(546.584269,345.045285),(511.517693,567.306546),(143.688165,967.436323),(227.594207,822.345773),(867.550643,801.070547),(328.668929,415.320103),(462.327246,614.674090),(585.247536,866.730458),(634.398506,842.430295),(256.187657,725.762789),(835.777832,432.161093),(624.414356,738.812776),(279.702800,724.775507),(712.703055,26.644034),(181.630075,251.075348),(138.709819,728.214344),(596.120633,650.227512),(295.520889,739.808798),(617.663834,523.115097),(562.154570,485.214477),(324.185643,890.823499),(900.534580,786.512890),(505.497589,485.782115),(653.243347,139.896094),(328.212411,909.431004),(865.658883,163.990242),(341.592098,490.073239),(902.803019,621.294897),(214.848746,615.506073),(647.938932,396.478821),(866.581421,786.648750),(124.693165,462.702053),(436.876261,420.214054),(202.510851,54.540095),(943.329151,764.665421),(539.754572,267.514794),(655.488920,440.289152),(54.027684,160.986508),(926.071267,707.271031),(300.882603,254.283677),(616.702035,166.541486),(418.273920,958.294133),(656.614725,321.076938),(579.589030,871.463471),(936.583011,227.527961),(267.942291,803.164432),(14.176711,392.635456),(265.866485,451.052973),(812.849510,468.377336),(505.593068,756.178661),(233.042757,45.347640),(23.693454,888.531677),(485.636791,77.721139),(49.518185,411.708058),(784.992169,350.400788),(665.991736,401.694204),(516.942274,84.265655),(359.988337,173.556999),(405.342594,939.577368),(45.020469,341.925605),(167.105328,312.962761),(145.090037,181.282040),(705.598217,410.956522),(632.335013,518.447727),(879.333858,137.928080),(274.626388,112.376615),(183.275720,298.319842),(0.908291,668.912512),(376.040981,50.426476),(80.620569,161.033150),(400.827265,746.612305),(562.727355,917.769539),(830.877961,922.715692),(91.326538,236.220555),(862.293059,136.347007),(578.146159,29.398387),(449.309768,723.236196),(210.680427,154.907986),(134.192717,843.015440),(673.355713,13.526575),(980.943520,947.982101),(125.903190,164.219240),(246.301943,126.811481),(833.131751,622.342924),(177.237957,913.752321),(783.376074,578.065222),(660.364626,346.103429),(495.834760,491.242586),(268.819121,587.161298),(727.463141,131.112180),(723.508305,305.609300),(160.510567,172.818073),(28.845496,371.190995),(327.726059,163.038213),(214.206434,1.081772),(176.564789,195.149954),(949.063872,302.467978),(359.369194,195.365815),(429.279459,192.500945),(817.708739,606.517416),(106.253266,601.084813),(184.582638,766.617892),(947.188242,680.417398),(257.860478,216.007363),(267.578696,985.323619),(347.119542,991.087001),(290.932919,507.630110),(163.905074,319.778415),(878.821105,491.631133),(482.816628,93.027538),(492.712904,659.381416),(288.177492,441.776777),(961.849395,647.546687),(637.142592,391.128854),(840.047632,454.851331),(997.646270,946.300898),(55.936144,182.228908),(712.918789,3.124386),(862.646306,970.779267),(219.131749,130.225001),(956.102886,566.251292),(121.312002,247.035805),(73.881401,285.217076),(566.814219,952.702506),(776.848209,49.630847),(45.730044,269.561113),(709.012264,333.907536),(711.337890,670.861658),(981.454223,348.480481),(61.990512,821.501854),(803.331813,59.636781),(767.802752,859.267957),(241.865689,480.721541),(862.392343,104.511994),(451.500807,81.524092),(234.736995,407.603693),(647.775384,356.048997),(654.639498,721.656785),(641.266073,221.453717),(674.359291,418.114281),(271.084564,720.089335),(687.675394,980.096828),(53.996871,399.013283),(650.958486,35.451094),(747.493765,712.948998),(856.952948,550.825577),(772.585779,624.755700),(410.093534,14.451467),(105.477240,272.485877),(118.963462,556.978047),(354.009969,353.700457),(964.581741,1.785352),(709.749454,619.221238),(723.442138,351.015527),(840.674955,397.801428),(769.129808,111.759519),(117.890763,456.805202),(91.856347,171.887635),(855.818485,742.814833),(207.338729,603.312250),(455.763831,64.291677),(154.137827,228.349610),(689.047377,564.231361),(242.801077,794.524617),(836.717238,361.764539),(351.502664,190.727207),(715.464997,316.084404),(192.512559,425.214451),(935.305642,915.954696),(776.229978,775.980597),(313.756125,545.359785),(887.740116,431.646888),(2.164987,979.596463),(603.534523,857.983472),(722.411296,810.873252),(461.295722,178.175126),(875.164929,615.433549),(406.524736,564.212306),(179.664910,649.325813),(358.736923,16.382148),(11.090353,710.239587),(207.109355,726.555349),(26.323990,399.621913),(151.769800,961.629632),(315.576609,927.999777),(737.610229,629.332734),(473.359562,625.350344),(60.979622,475.524549),(604.946806,664.514145),(333.508021,327.358102),(475.387397,794.803744),(505.533228,350.552325),(410.237292,912.057964),(914.764631,589.902203),(561.383778,273.501553),(606.284351,572.474130),(983.741140,813.393705),(299.029479,10.065130),(213.015618,450.799279),(971.694763,528.592228),(378.799056,709.304991),(157.924962,852.158618),(334.655335,218.904584),(327.683167,939.602142),(883.418729,661.191189),(266.960243,358.806125),(455.994932,772.493471),(709.358450,866.232224),(684.551435,624.123081),(456.134427,245.935212),(897.624635,62.418778),(818.409342,881.365774),(875.812483,117.438821),(891.430905,88.828101),(568.238100,863.125667),(617.420330,947.037156),(572.430657,775.345292),(799.195774,907.085993),(994.249876,126.878941),(846.688134,877.668604),(788.070129,113.648376),(236.474729,244.065060),(886.141848,945.833179),(110.297284,570.693283),(569.956260,566.431711),(816.628495,467.580894),(628.850489,635.037837),(348.946668,504.662972),(752.476658,240.377572),(593.491074,320.714757),(103.503238,210.911403),(267.751913,675.933896),(986.256695,66.947687),(583.019889,980.506570),(193.826627,429.708022),(858.175174,981.896756),(543.356399,94.649903),(225.961816,429.498247),(40.483082,336.259100),(0.191529,610.439342),(902.690812,816.820023),(78.020236,531.541301),(451.857860,426.966903),(36.204272,204.334518),(667.344476,629.695346),(525.049275,770.847714),(840.606749,792.801189),(446.781610,826.863443),(859.748875,29.801499),(807.370013,53.575502),(459.509521,665.545187),(35.472258,2.865920),(760.195090,261.434074),(432.364166,800.678172),(597.693174,432.555695),(411.117513,500.383985),(249.375718,489.137748),(31.925286,701.233579),(916.104652,68.129558),(905.568096,583.449128),(697.824905,430.617371),(354.296842,538.431653),(223.418560,801.078453),(365.295096,83.167434),(830.879951,172.665109),(136.742936,290.389472),(838.210297,172.215194),(293.255392,598.405386),(433.649267,725.619559),(399.083558,31.342441),(158.175253,810.201071),(531.726427,407.550972),(299.338819,563.651713),(108.784550,215.443471),(631.781271,14.352646),(798.892599,329.606175),(444.970018,153.189440),(868.037829,668.388577),(954.267893,233.332925),(751.556012,785.147844),(405.998034,888.298948),(75.537316,244.208330),(60.514141,368.792708),(842.613717,494.163409),(94.412266,241.697274),(525.505850,252.587520),(51.898344,57.232277),(660.138491,351.237164),(620.883989,768.923042),(566.680635,252.665260),(783.275688,365.573233),(582.271436,228.245705),(518.762673,450.309264),(896.634282,473.030566),(683.642188,648.190294),(258.178410,89.640222),(536.489242,333.715726),(333.848552,597.003383),(702.508435,176.462269),(91.166792,796.920701),(418.159543,616.672642),(49.508220,470.057887),(673.904919,709.646711),(821.295051,294.788908),(478.569753,387.975685),(547.454169,261.845440),(753.548918,129.725604),(490.091146,272.311592),(580.034868,386.725428),(745.342158,263.677057),(36.915722,2.520568),(353.317279,571.404964),(337.236294,687.165832),(168.408347,39.744728),(863.628101,259.575139),(836.665429,281.787643),(876.247782,886.173649),(751.845531,550.152701),(595.820360,573.140581),(844.941609,74.390113),(961.116266,392.395778),(336.235553,714.665184),(522.121382,826.326699),(986.976776,102.156250),(213.052127,732.318933),(365.833306,247.967848),(735.839501,719.150585),(819.372812,73.075795),(406.316417,987.781159),(112.820523,269.944517),(247.356298,949.485952),(551.732159,123.604079),(835.659601,303.577690),(673.756779,431.479961),(876.718271,518.698388),(505.870074,837.834537),(911.094166,842.105627),(552.499721,433.215548),(668.432326,539.476496),(535.371798,881.484453),(271.795429,901.205105),(129.452301,7.634931),(620.355690,948.825113),(80.710726,26.672106),(936.606272,193.531250),(296.616622,183.962569),(143.017202,848.348782),(307.566648,978.676803),(151.926471,981.323428),(410.156764,28.644742),(500.021816,916.026839),(866.479279,411.115982),(758.132466,418.978999),(844.331530,426.564791),(958.455496,379.703327),(308.049244,230.250925),(280.908432,437.501545),(237.885856,901.264121),(386.326658,318.596582),(927.936227,322.932929),(512.127831,224.552849),(506.895499,655.145034),(72.901631,814.462147),(633.821837,224.828102),(795.785574,43.978600),(253.472844,295.807390),(960.005439,119.952123),(706.923372,718.137904),(538.931122,551.254901),(144.702695,497.386617),(930.958229,452.751939),(727.637542,211.866660),(890.253485,965.523398),(113.130781,276.580142),(284.119979,41.067007),(599.513072,796.247811),(265.619857,106.408570),(451.392844,338.521488),(920.870717,85.214680),(563.349590,716.656291),(129.193280,816.822434),(12.463680,89.198719),(936.774557,719.387052),(807.336623,475.705678),(270.641952,952.039318),(973.092296,201.600181),(404.791258,700.729838),(413.466841,295.044742),(666.253235,526.597622),(571.624884,950.373215),(567.664629,171.137956),(746.621025,833.284486),(277.546526,198.013869),(171.805973,198.417242),(283.228549,735.155563),(915.073533,412.421829),(551.977997,927.537213),(501.620548,488.752553),(646.924265,308.957170),(964.458232,917.566217),(260.996489,937.550527),(119.166398,665.787746),(638.280365,532.633238),(960.832488,304.533600),(59.230859,532.457372),(254.906814,626.895489),(703.595328,1.527838),(460.179975,981.141854),(199.541707,631.985948),(179.559095,482.770256),(367.141511,94.632628),(895.192085,919.119508),(22.169841,396.812633),(407.872061,669.094106),(705.769804,372.330293),(586.660323,966.766292),(309.880821,705.826720),(632.554038,948.161186),(238.459958,593.386525),(252.694785,297.690818),(125.843897,507.601599),(924.586307,829.439225),(509.129437,384.766281),(810.581078,708.671145),(16.752229,990.140173),(191.441401,383.893740),(84.772801,86.633486),(303.013248,106.942641),(483.446119,710.885310),(776.036747,189.215922),(83.215603,362.697069),(155.982214,393.096423),(68.523789,788.536251),(341.257609,306.983747),(381.922777,593.952394),(604.674565,507.766673),(101.553992,529.260872),(337.205897,610.683430),(914.027153,147.786975),(319.354574,930.779382),(137.927147,510.795975),(314.673122,222.699948),(597.429461,617.686370),(329.642589,80.875579),(328.571680,105.679336),(270.091502,411.787282),(468.376405,426.073715),(804.883706,536.900194),(214.609967,146.141314),(843.883941,596.532743),(740.093708,448.558505),(104.299416,841.647701),(977.819377,441.505314),(452.331131,891.846530),(589.292288,771.685705),(822.625911,727.219436),(282.481680,137.299033),(949.919383,879.911141),(754.985403,279.561972),(960.786721,83.557082),(385.241308,230.878222),(495.344365,853.617713),(656.951937,300.228071),(390.517906,871.561904),(446.369386,234.401847),(468.094647,186.463093),(682.960352,572.394063),(28.110794,660.779729),(13.899376,480.441925),(552.626258,603.191664),(252.127629,375.252170),(330.411100,534.609310),(512.551203,280.330483),(414.520450,267.536605),(559.892455,375.307170),(351.093688,945.133764),(606.185392,846.438053),(798.751477,263.137329),(146.666123,189.269383),(134.699233,593.035509),(423.671229,602.793880),(779.498602,106.631582),(175.187943,807.609396),(767.411311,189.087319),(288.051321,320.037569),(792.278983,540.178951),(695.289739,122.690083),(74.788260,207.840941),(403.020566,489.308710),(475.377546,962.913021),(864.615881,826.471234),(908.046785,470.801273),(672.909287,706.798261),(733.938602,819.575410),(896.067644,868.637836),(412.610918,319.738873),(471.431715,192.109520),(426.370455,646.619658),(999.718917,193.781765),(835.706977,287.770237),(513.819335,627.985959),(827.949188,209.109073),(750.676042,902.737447),(416.950015,153.696607),(392.046157,892.327561),(116.609628,256.662038),(718.798795,24.656412),(727.463310,391.708081),(731.454673,461.401913),(211.283491,627.522317),(330.039748,623.894409),(947.261190,801.471463),(816.003928,373.631644),(448.091120,815.722844),(567.413410,283.798096),(103.493081,81.232744),(911.784056,931.442269),(290.341817,662.460097),(834.179717,707.291832),(816.156704,226.225874),(599.619393,932.766332),(482.887911,318.418188),(957.422744,210.351221),(710.126269,688.877418),(671.753134,921.409760),(316.399734,1.792881),(545.304168,263.660923),(803.264344,361.308096),(637.292568,251.355464),(177.030940,204.705977),(535.153560,280.524022),(285.938722,446.937616),(211.966290,576.280539),(109.397713,46.146006),(283.572371,925.554417),(272.371880,883.191764),(858.320749,755.259791),(201.609952,815.743493),(965.611013,911.736221),(504.620910,637.364147),(833.145980,821.020645),(639.157028,378.450147),(84.681567,442.421372),(739.758243,721.974135),(693.776836,916.789183),(926.680113,228.930396),(197.313204,212.618834),(675.868011,409.279495),(788.899374,785.265724),(455.425501,72.471744),(710.820141,727.797381),(955.663509,569.140890),(483.057172,157.273460),(384.884383,448.668185),(69.009680,889.505294),(86.032331,902.155660),(710.525938,725.189360),(280.605806,795.207505),(167.610731,20.364048),(517.181640,861.387567),(937.153232,443.861752),(90.317963,134.466435),(656.480587,766.185974),(543.745930,445.379960),(551.451698,999.171432),(517.851705,262.271839),(726.968813,473.515213),(831.412729,210.025984),(630.788673,216.297111),(658.694170,699.798352),(105.802404,744.726501),(601.954012,816.328342),(469.915861,882.559818),(611.535847,637.526592),(902.923866,128.717487),(498.914159,840.077097),(572.579239,589.232121),(974.543533,229.059826),(355.418095,518.289463),(674.439786,906.869793),(517.460894,192.291490),(169.141632,244.429706),(665.806703,0.554360),(454.455691,296.595375),(216.851472,113.149860),(996.393727,322.653876),(857.876362,598.347738),(138.982218,327.792222),(480.907555,750.518065),(965.318813,383.831422),(879.235552,464.232972),(223.908519,451.814790),(53.465093,198.452051),(680.874617,408.883188),(716.741515,355.314402),(315.752981,234.202408),(547.605892,484.894613),(478.632115,213.412595),(485.448973,933.087806),(510.007970,702.300445),(46.237666,506.401696),(24.954320,904.114028),(104.749435,163.936538),(231.906249,585.656990),(914.454603,197.225062),(969.488412,793.690154),(661.458034,193.396930),(245.504944,714.923127),(391.848982,926.379561),(123.806315,108.590496),(281.693963,439.559296),(342.792905,829.299855),(924.453909,821.425020),(42.712450,409.902882),(754.512825,552.720419),(112.203326,800.750492),(59.122116,137.157647),(704.864519,163.871551),(301.094185,936.770768),(749.528541,215.548787),(133.995830,719.016952),(9.238941,795.453865),(912.413883,254.743885),(510.376991,304.262864),(181.123446,634.183306),(412.853360,462.817409),(73.742602,755.646265),(292.117264,998.196511),(577.071284,334.829714),(408.099392,331.584109),(887.550134,520.302718),(132.334600,946.672250),(657.460365,837.199119),(110.543800,958.554550),(773.969887,860.072340),(174.103336,907.965717),(579.089292,183.342278),(703.419581,491.503174),(438.086163,213.796572),(795.766038,619.209609),(847.979878,208.619398),(82.027018,921.722481),(964.265662,374.144282),(919.918991,541.336946),(708.973997,328.018382),(872.921054,596.524130),(848.321101,5.255654),(543.196379,505.781465),(842.454772,653.740179),(464.336014,616.424659),(513.812518,638.439351),(524.390375,92.901811),(821.781628,227.809956),(584.404985,259.867791),(441.606528,380.171023),(879.077400,289.586407),(588.790420,961.104419),(211.308887,553.056082),(335.248701,131.227877),(94.393027,44.222697),(459.246260,967.314081),(640.746827,307.567360),(972.569735,183.943206),(813.348825,815.024507),(837.683384,277.684839),(431.449165,351.495903),(916.124190,955.839540),(444.397713,737.905818),(183.649496,28.802698),(997.773609,625.256025),(408.973721,876.851010),(914.842431,997.764141),(837.955428,126.151318),(550.820223,173.204129),(257.379195,645.213250),(217.426826,716.625455),(612.527331,858.173653),(24.192815,585.097065),(42.116859,837.541640),(400.121572,879.800243),(115.226479,831.570737),(231.296145,31.350669),(787.410277,675.693859),(769.256487,971.059773),(704.496557,767.030096),(596.315797,113.470277),(643.881105,511.158228),(111.234417,481.836532),(637.309546,662.054640),(655.040662,894.688741),(307.267890,872.467488),(611.314195,919.795221),(730.641141,635.507010),(504.892286,772.758000),(473.048650,905.013858),(652.558243,588.275129),(736.584595,883.854389),(619.625798,523.994872),(559.548247,388.882284),(495.054644,264.044804),(155.912379,91.370441),(377.515081,799.793485),(602.528669,488.749498),(281.630017,239.838215),(150.804138,936.670678),(134.526955,458.072028),(809.138166,745.841151),(377.867249,539.779307),(381.348161,882.759535),(312.537306,854.396811),(787.773393,965.095549),(442.671939,524.357988),(848.949938,62.297736),(48.352859,408.498185),(451.180021,543.407504),(672.542989,607.092400),(634.777945,50.058069),(406.885884,237.306614),(538.807567,688.515901),(477.144829,689.611704),(625.186579,611.671784),(147.683732,434.324744),(357.512935,525.550980),(974.104051,738.861096),(408.310515,286.641357),(593.257906,196.083908),(251.736906,35.929845),(720.441895,100.686844),(98.227581,768.794754),(509.185028,549.407602),(312.202258,181.728017),(156.500002,946.980203),(231.786085,563.385886),(184.286817,770.593652),(251.901787,661.431645),(460.205356,877.088366),(273.103430,607.889089),(311.413110,630.616364),(133.440069,285.517161),(369.477460,541.750584),(572.158518,962.735366),(737.834491,823.895425),(998.665210,458.276386),(924.582268,96.892791),(227.071141,433.767297),(646.300393,539.273398),(615.495313,802.800395),(486.253601,847.281399),(366.186281,670.540417),(617.875051,618.088068),(331.972063,78.080407),(495.176434,605.075492),(685.969496,806.589545),(235.691856,819.409565),(92.106705,605.169316),(361.160148,664.265224),(567.904682,98.994639),(488.160648,566.569892),(557.271025,412.742915),(663.462683,784.342165),(846.510212,309.763076),(323.615564,462.005525),(112.563471,809.869165),(309.286924,478.749752),(480.409582,927.161975),(96.837821,812.381645),(5.242381,592.014255),(417.457137,691.211877),(398.603799,653.148993),(510.621441,490.710504),(258.318309,871.781589),(154.975727,826.222991),(970.776228,643.136375),(392.792882,528.047253),(55.879290,56.255565),(312.389418,902.389502),(366.018641,636.004981),(364.395027,478.582112),(445.874146,673.681951),(957.331864,926.283728),(600.843925,54.169684),(738.665372,606.086307),(646.183939,156.122508),(297.298183,44.787738),(809.271501,807.919625),(535.498242,67.589809),(679.701213,690.473969),(893.812800,650.477441),(333.610344,286.605681),(178.524693,389.489635),(342.861246,490.914111),(291.879136,708.879887),(126.919092,656.274163),(187.461998,572.793238),(329.956114,144.793862),(499.076965,930.800039),(198.963546,237.742337),(536.886345,845.147486),(393.864845,834.184528),(889.935223,203.136345),(642.104152,425.433465),(270.726155,321.805365),(115.907434,164.538954),(972.282806,449.517779),(451.144636,150.807499),(839.007413,794.005882),(641.721610,130.886549),(502.885768,768.640702),(787.160713,690.347766),(341.433940,117.116826),(835.141628,840.510905),(47.916865,34.105175),(78.253242,584.803209),(879.252660,472.118087),(418.987737,769.187883),(675.254432,61.091889),(194.621348,945.980587),(382.897254,310.528783),(110.519542,355.180060),(760.046562,561.664178),(505.987559,599.053975),(355.670060,147.709168),(729.940524,858.555828),(916.349870,517.101236),(548.903594,257.783809),(634.218063,384.045222),(98.294714,682.134927),(418.150397,176.547956),(266.938136,297.403057),(648.666043,685.925873),(66.590940,323.920475),(747.017763,261.212289),(269.901062,129.915016),(571.741071,380.420604),(485.095076,331.787632),(942.084782,991.082636),(930.841607,297.754842),(138.791803,660.782131),(156.310669,55.141673),(177.883367,705.214263),(312.925483,812.101429),(89.259485,411.220197),(494.236356,507.409883),(587.768153,761.174492),(804.812940,236.434196),(447.100365,871.403880),(560.354671,194.118127),(132.616168,830.255733),(324.033143,704.357240),(210.676337,809.128220),(36.144872,152.761119),(800.210855,966.986480),(450.515960,939.002659),(627.768610,606.826629),(994.144332,805.651977),(312.040892,307.069815),(617.753405,401.300378),(718.290012,111.989761),(908.710260,306.058165),(873.164253,713.523200),(542.492361,320.264617),(584.927080,102.847031),(514.382744,717.543248),(933.102764,838.415887),(421.900488,143.779100),(647.544107,458.045361),(296.540219,447.754962),(425.031840,747.056179),(386.757620,52.800450),(353.882808,380.901952),(858.452426,665.923700),(687.971767,476.205831),(67.224077,406.261779),(588.195592,975.934338),(712.319944,461.359844),(689.457538,254.812304),(781.624461,274.384617),(357.659335,296.007205),(991.927866,290.762099),(134.423092,413.828354),(434.541200,781.967199),(871.873715,731.081419),(229.722160,296.905555),(478.137597,616.479780),(349.706004,832.020405),(997.381733,208.158430),(497.944105,685.353499),(684.364261,565.168183),(91.615278,272.559852),(541.102520,803.935222),(733.919697,230.560058),(58.747525,515.544158),(504.944675,416.406861),(811.551363,496.872541),(707.168960,945.974454),(910.700895,141.710159),(727.941653,782.574609),(872.791578,957.663813),(79.480163,350.929174),(574.143593,429.186167),(182.949579,571.525325),(637.344597,680.893684),(256.878825,321.708858),(246.061866,348.494102),(594.268710,787.164387),(152.429324,328.188406),(17.724444,211.176849),(843.732564,522.669120),(627.583710,655.283926),(19.541660,334.752670),(601.258381,930.242555),(476.462829,329.200033),(712.817164,349.254406),(286.863845,792.297328),(700.183581,861.007439),(221.483495,883.133160),(432.532764,858.828092),(564.026844,689.411588),(180.536950,810.088710),(37.905690,774.805660),(597.253096,190.335014),(102.994066,614.977540),(401.511864,946.726631),(137.646659,29.095573),(602.010557,157.188320),(363.848243,203.268937),(87.430874,840.311072),(532.468970,800.248038),(189.565478,819.332816),(592.545366,889.749059),(680.340254,814.028861),(772.882219,112.873018),(672.856953,336.909062),(802.284606,853.393902),(146.997772,840.190297),(628.199562,744.250868),(30.525310,731.193629),(359.228408,432.037174),(677.920259,496.875068),(461.132747,279.930816),(654.063388,824.980991),(483.199753,741.494262),(665.292063,15.668723),(541.742300,854.857541),(835.001538,134.287666),(744.606600,515.341792),(948.316527,517.488819),(628.214810,621.173479),(854.397881,430.499416),(474.567381,1.395653),(270.689712,102.766943),(745.646521,301.215023),(833.960572,104.874929),(733.252197,511.880831),(601.749997,194.384945)
