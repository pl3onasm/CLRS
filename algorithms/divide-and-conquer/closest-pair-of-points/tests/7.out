Closest distance: 1.484202
Found between (35.472258,2.865920) and (36.915722,2.520568)
